### PR TITLE
data: yashwanth-3000

### PR DESCRIPTION
**Pull request by yashwanth-3000**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8629"></a>

### CODE REVIEW

**Review by accodes21 — CHANGES_REQUESTED** [review]
Please change the the filename to that of your username i.e ```yashwanth-3000```

File: data/Yashwanth.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    {
+      "name": "Gitub link",
+      "url": "https://github.com/yashwanth-3000",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Linkedin profile ",
+      "url": "https://www.linkedin.com/in/pyashwanthkrishna/"
+      "icon": "FaLinkedin"
+    },
```
Comment (sital002) [inline data/Yashwanth.json:31]:
```suggestion
    }
```
This is a trailing comma which isnot required

**Review by sital002 — CHANGES_REQUESTED** [review]
Your filename must match your github username. It should be yashwanth-3000.json instead of Yashwanth.json

File: data/Yashwanth.json (hunk 1 above)
Comment (yashwanth-3000) [reply data/Yashwanth.json]: have changed this user name and comma is that fine?

Comment (sital002) [reply data/Yashwanth.json]: > have changed this user name and comma is that fine?

No the filename must be yashwanth-3000.json It must have the .json file extension.

File: data/yashwanth-3000 [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Yashwanth krishna",
+  "bio": "Trying to contribute to open source while learning. Learing in Public ",
+  "tags": ["Machine learning","DSA"],
+  "social": [
+    {
+      "icon": "Fatwitter",
+      "Url": "https://twitter.com/Yashwanthstwt",
```
Comment (SaraJaoude) [inline data/yashwanth-3000:8]: remove comma:
```suggestion
      "Url": "https://twitter.com/Yashwanthstwt"
```

File: data/yashwanth-3000 [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    },
+    {
+      "icon": "Falinkedin",
+      "url": "https://www.linkedin.com/in/pyashwanthkrishna/",
```
Comment (SaraJaoude) [inline data/yashwanth-3000:12]: remove comma:
```suggestion
      "url": "https://www.linkedin.com/in/pyashwanthkrishna/"
```

File: data/yashwanth-3000 [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Gitub link",
+      "url": "https://github.com/yashwanth-3000",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Linkedin profile ",
+      "url": "https://www.linkedin.com/in/pyashwanthkrishna/"
```
Comment (SaraJaoude) [inline data/yashwanth-3000:29]: add comma:
```suggestion
      "url": "https://www.linkedin.com/in/pyashwanthkrishna/",
```

**Review by SaraJaoude — APPROVED** [review]
I have made a few corrections to the JSON and can now merge this PR

**Comment by sital002** [comment]
> I have made a few corrections to the JSON and can now merge this PR

The filename doesnot have .json extension thats why the test was also failing.

**Comment by eddiejaoude** [comment]
Good spot, thank you 👍  I will fix

**Comment by yashwanth-3000** [comment]
Thank you accodes21,sital002 and SaraJaoude for helping me out and Eddie ofc :)

**Pull request merged**